frontend/src/Layout/Router.jsx

```
@@ -0,0 +1,71 @@
+        children: [
+            {
+                path: "/studentDashBoard",
+                element: <StudentDashBoardHome />,
+            },
+            {
+                path: "messages", // Change this from "/messages" to "messages"
+                element: <Message></Message>
+            },
+            {
+                path: "result", // Change this from "/messages" to "messages"
+                element: <Result></Result>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)